config/default/kustomization.yaml

```diff
@@ -18,9 +18,9 @@ bases:
 - ../manager
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in 
 # crd/kustomization.yaml
-#- ../webhook
+- ../webhook
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
-#- ../certmanager
+- ../certmanager
```

Comment:
Is there a reason we are also enabling certmanager in this PR? That seems to be what is failing the integration tests.

Reply:
This provides certificates and related resources for webhook api serves in https(which is mandatory). Integration test is now failing because previously cert-manager crd resources were applied(certificate, issuer, etc. crds) and I guess now it is not.